src/js/actions/search/adobestock.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2014 Adobe Systems Incorporated. All rights reserved.
```

Comment:
The rule is: whenever you make a new file, make sure the copyright notice uses the current year. So, this should be 2016 instead of 2014.

Aside: for brevity, I probably would have just named this `stock.js`, but if you really like `adobestock.js` then OK!